src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Components/Startup.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.AspNetCore.Components.Builder;
+
+namespace RazorComponentsWeb_CSharp.Components
+{
+    public class Startup
+    {
+        public void Configure(IComponentsApplicationBuilder app)
+        {
+            app.AddComponent<App>("app");
+        }
+    }
+}
```

Comment:
Looking forwards to getting rid of this file, pending @javiercn's startup improvements.